contracts/DCNT721A.sol

```diff
@@ -338,4 +339,43 @@ contract DCNT721A is ERC721A, DCNT721AStorage, Initializable, Ownable, Splits {
     presaleStart = newStart;
     presaleEnd = newEnd;
   }
+
+  // @notice update the registration with the operator filter registry
+  // @param enable whether or not to enable the operator filter
+  // @param operatorFilter the address for the operator filter subscription
```

Comment:
```suggestion
  /// @notice update the registration with the operator filter registry
  /// @param enable whether or not to enable the operator filter
  /// @param operatorFilter the address for the operator filter subscription
```

Reply:
need 3 `///` for NATSPEC

https://docs.soliditylang.org/en/v0.8.17/natspec-format.html#documentation-example